tests/qos/files/qos_params.th2.yaml

```diff
@@ -184,6 +184,34 @@ qos_params:
                 cell_size: 208
             hdrm_pool_wm_multiplier: 4
             cell_size: 208
+        topo-dualtor-aa-56:
+            50000_40m:
+                pcbb_xoff_1:
+                    dscp: 3
+                    ecn: 1
+                    pg: 3
+                    pkts_num_trig_pfc: 19939
```

Comment:
Is this value equal with the result of "qos_param_generator.py" for same asic and topology?

Reply:
Yes.